axelrod/player.py

```diff
@@ -134,6 +134,12 @@ def strategy(self, opponent):
         """This is a placeholder strategy."""
         raise NotImplementedError()
 
+    def dual_strategy(self, opponent):
+        from copy import deepcopy
+        fresh_opponent = deepcopy(opponent)
+        self.original.play(fresh_opponent)
```

Comment:
I believe this method is not actually necessary (the transformer does not use it). Can it just be removed?